plugin/license.vim

```diff
@@ -31,12 +31,8 @@ function! InsertLicense(lic, stub)
     if l:which > -1
         execute '0read ' . l:files[l:which]
         execute "'[,']" . 's/{YEAR}/\=strftime("%Y")/e'
-        if exists('g:license_author')
-            execute "'[,']s/{AUTHOR}/" . g:license_author . '/e'
-        endif
-        if exists('g:license_email')
-            execute "'[,']s/{EMAIL}/" . g:license_email . '/e'
-        endif
+        execute "'[,']s/{AUTHOR}/" . get(g:, 'license_author', executable('git') ? system('git config --global user.name') : $USER) . '/e'
```

Comment:
What does the `$USER` here mean?

Reply:
i have changed it to expand('$USER'). in `help expand`, there are
> expand() can also be used to expand variables and environment variables that are only known in a shell.  But this can be slow, because a shell may be used to do the expansion.  See |expr-env-expand|.  The expanded variable is still handled like a list of file names.  When an environment variable cannot be expanded, it is left unchanged.  Thus ":echo expand('$FOOBAR')" results in "$FOOBAR".

in unix-like OS, it gets the value of $USER, in Windows, it gets the value of %USER%.

Reply:
Awesome, thanks for explaining :+1: